relayer/.env.devnet.example

```diff
@@ -1,11 +1,11 @@
-KEY_PAIR=
-RELAYER_RECIPIENT=
-LOOK_UP_TABLE=
-LOCAL_TEST_ENVIRONMENT=false
-REDIS_ENVIRONMENT=LOCAL
-NETWORK=DEVNET
-RPC_URL=
-# DB_PORT=25061
-# HOSTNAME=
-# PASSWORD=
-# USERNAME="default"
\ No newline at end of file
+export KEY_PAIR=
+export RELAYER_RECIPIENT=
+export LOOK_UP_TABLE=
+export LOCAL_TEST_ENVIRONMENT="false"
+export REDIS_ENVIRONMENT="LOCAL"
+export NETWORK="DEVNET"
+export RPC_URL=
+# export DB_PORT="25061"
+# export HOSTNAME=
+# export PASSWORD=
+# export USERNAME="default"
```

Comment:
leftover comments?

Reply:
added an inline comment - these will need to be used if folks want to access a remote DB